src/media/js/main.js

```diff
@@ -151,7 +151,9 @@ function(_) {
         // are in a packaged app, wait for the iframe to be loaded, otherwise
         // we are using the direct installer and we just need to wait for the
         // normal loaded event.
-        var event_for_apps = window.location.protocol === 'app:' ? 'iframe-install-loaded' : 'loaded';
+        var is_packaged_app = window.location.protocol === 'app:';
+        var is_iframed_app = window.top != window.self;  // Crude, but enough for our needs.
```

Comment:
we have a [`capabilities.yulelogFxA`](https://github.com/mozilla/marketplace-core-modules/blob/3f01908/capabilities.js#L34) which does the same iframe check but also checks a waffle.

and we do the same iframe check again in the [`webactivities` module](https://github.com/mozilla/fireplace/blob/095c0c8/src/media/js/webactivities.js#L75).

perhaps you could consolidate? might need to check it again in more places in the future.